website/cue/reference/components/enrichment_tables/file.cue

```diff
@@ -0,0 +1,141 @@
+package metadata
```

Comment:
This isn't a component and it shouldn't follow the component behavior or patterns. We also use "component" to refer to actual nodes in the processing graph. We should just add a root level file in the [`reference` folder](https://github.com/vectordotdev/vector/tree/master/website/cue/reference) called `enrichment_tables.cue`.